test/modifications.js

```diff
@@ -0,0 +1,100 @@
+import Color from "../src/index.js";
+import { check } from "./util.mjs";
+
+export default {
+	name: "Color modification tests",
+	description: "These tests modify one or more coordinates and check the result.",
+	run (func) {
+		return func();
```

Comment:
That is never needed, you can just have a `run` on each individual test. :)

Reply:
Fixed, that's way better.